ms-authentication/routes/controller_test.go

```diff
@@ -34,10 +35,10 @@ func TestController_AddAllRoutes(t *testing.T) {
 			mockAppService := &mocks.ApplicationService{}
 			if !tt.failAddRoute {
 				mockAppService.On("AddRoute", mock.Anything, mock.Anything, mock.Anything, mock.Anything).
-					Return(nil)
+					Return(nil).On("LoggingClient").Return(logger.NewMockClient())
 			} else {
 				mockAppService.On("AddRoute", mock.Anything, mock.Anything, mock.Anything, mock.Anything).
-					Return(fmt.Errorf("fail"))
+					Return(fmt.Errorf("fail")).On("LoggingClient").Return(logger.NewMockClient())
```

Comment:
For readability, please don't chain these. The second mock is not so apparent as when we just have
mockAppService.On("LoggingClient").Return(logger.NewMockClient())
On its own line.

Reply:
agreed, moved to new line

Reply:
Not what I intended... ;-) See new comments.